assets/messages.yml

```diff
@@ -740,6 +740,30 @@ messages:
         You are currently using a *non-authorized* version of Minecraft. If you wish to purchase the full game, please visit the [Minecraft Store|https://www.minecraft.net/store/minecraft-java-edition].
         We will not provide support for pirated versions of the game, these versions are modified and may contain malware.
 
+        %quick_links%
+      fillname: []
+  provide-affected-versions:
+    - project:
+        - bds
+        - mc
+        - mcd
+        - mce
+        - mcl
+        - mcpe
+        - mctest
```

Comment:
Including `mctest` is not necessary and probably breaks the build because it does not exist under `project_id`?

Reply:
> Or is that useful when Arisa starts using these messages and is tested on MCTEST?

Yeah, that's the reason